internal/config/config.go

```diff
@@ -53,12 +54,14 @@ func RegisterConfigFile() error {
 
 	slog.Debug("Configuration file loaded", "config_path", configPath)
 	viperInstance.Set(ConfigPathKey, configPath)
+	viperInstance.Set(UUIDKey, uuidLibrary.Generate(configPath))
```

Comment:
Since we are hashing the config path, does that not mean that we will have the same UUID for each instance in a cluster of instances, assuming all instances use the default Agent conf path?

Reply:
Looks like we don't send this out, so I suppose it doesn't matter ıf there's a collision between instances

Reply:
I think it does matter, since we set this to be the instance id eventually. Will update

Reply:
can we make sure this is unique since it looks like it's being use in the auth header